src/storage.rs

```diff
@@ -68,16 +71,34 @@ impl Default for AccountBasic {
     }
 }
 
-/// EVM Code, currently mapping to REVM's [ByteCode::LegacyAnalyzed].
-// TODO: Support raw legacy & EOF
+/// Analyzed legacy code.
 #[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
```

Comment:
Is `PartialEq` needed here 🤔

Reply:
Yes, it is required for `EvmCode` -> `EvmAccount` -> `EvmStateTransitions` -> `PevmTxExecutionResult`. It won't be "required" anymore after #202 but comparing bytecodes still sounds convenient. I guess we can also implement more performant `PartialEq`s by comparing only the raw bytecodes, but it's out-of-scope here :pray:.

Reply:
We can remove the `Default`s though, which is also error-prone. Anything that can have an `EvmCode` like `EvmAccount` should use `None` instead of a default code (that is probably invalid anyway).

Reply:
Thank you, makes sense.